src/components/AddTodoForm.js

```diff
@@ -30,4 +31,8 @@ function AddTodoForm({ onAddTodo }) {
   )
 }
 
+AddTodoForm.propTypes = {
+  onAddTodo: PropTypes.func,
```

Comment:
You might want to make this prop a `required` prop as it is actually a required one so you get error if not passed.

`onAddTodo: PropTypes.func.isRequired`

Reply:
Oh, thanks for pointing that out, @eimaam. I'll add that!